todolist/file_store_test.go

```diff
@@ -8,13 +8,18 @@ import (
 
 func TestFileStore(t *testing.T) {
 	assert := assert.New(t)
-	store := &FileStore{FileLocation: "todos.json"}
-	todos, _ := store.Load()
-	assert.Equal(todos[0].Subject, "this is the first subject", "")
-}
+	// load the fixutre data
+	list := SetUpTestMemoryTodoList()
+	testFilename := "TestFileStore_todos.json"
+	store := &FileStore{FileLocation: testFilename}
+	defer testFileCleanUp(testFilename)
+	list.FindById(2).Subject = "this is an non-fixture subject"
+	// save the fixture data
+	store.Save(list.Todos())
 
-func TestSave(t *testing.T) {
-	store := &FileStore{FileLocation: "todos.json"}
-	todos, _ := store.Load()
-	store.Save(todos)
+	// reload the fixture data
+	store1 := &FileStore{FileLocation: testFilename}
+	todos, _ := store1.Load()
+	assert.Equal(todos[1].Subject, "this is the first subject", "")
+	assert.Equal(todos[0].Subject, "this is an non-fixture subject", "")
```

Comment:
2 comments about this test:

1.  I like tests formatted using [Thoughtbot's recommendations](https://robots.thoughtbot.com/how-we-test-rails-applications), so something along the format of:
```
setup

exercise

assertions
```

The main point is that the exercise has blank lines above and below it.  In this case, I think `store1.Load()` is probably the exercise.  The stuff above it is setup.

2. not sure if the comments are needed.